primefaces/src/main/resources/META-INF/resources/primefaces/idlemonitor/1-idlemonitor.js

```diff
@@ -98,11 +98,13 @@ PrimeFaces.widget.IdleMonitor = PrimeFaces.widget.BaseWidget.extend({
      * @inheritdoc
      */
     destroy: function() {
-        this._super();
-
         if (this.cfg.multiWindowSupport) {
             clearInterval(this.timer);
         }
+        $(document).off('idle.idleTimer active.idleTimer');
```

Comment:
what happens here with multiple timers on the view?

Reply:
Great catch I need to be using the ID.  PR updated.